Chemistry/SimulateHubbardHamiltonian/HubbardSimulation.qs

```diff
@@ -22,7 +21,7 @@ namespace Microsoft.Quantum.Chemistry.Samples.Hubbard {
         mutable spinUpIndices = new Int[nFilling / 2];
         
         for (idx in 0 .. nFilling / 2 - 1) {
-            set spinUpIndices[idx] = idx;
+            set spinUpIndices w/= idx <- idx;
```

Comment:
this also looks like a case where an initializer function from the array module should be used...

Reply:
Definitely agreed. My goal with this PR was to convert things literally, but I think this snippet could be refactored to use an array function pretty readily.

Reply:
Looking again, that array isn't used anywhere? I've asked @guanghaolow and will update or fix accordingly. Thanks for catching this as well!

Reply:
After discussing with @guanghaolow, I believe this was a minor bug, and have fixed it accordingly, and have added him as a reviewer for the PR. Thanks again for catching this, @bettinaheim!